docs/integrations/linode/gslb.mdx

```diff
@@ -0,0 +1,322 @@
+
+ngrok's GSLB even works on top of Linode's internal load balancing mechanism, NodeBalancer, for additional global resiliency and latency improvements.
+
+:::caution This how-to guide requires:
+
+1. An [ngrok account](https://ngrok.com/signup).
+1. An account with [Linode](https://login.linode.com/signup).
+1. Three Ubuntu 24.04 virtual machines (VMs) in three globally-distributed regions (e.g. Atlanta, Osaka, and Frankfurt):
+   - Hostnames should be unique, ideally using the location of the datacenter (e.g. `atlanta`, `osaka`, and `frankfurt`).
+   - [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on each VM.
+
+:::
+
+## **Step 1**: Reserve your ngrok domain {#reserve-domain}
+
+Throughout this guide, you'll use the [ngrok API](https://ngrok.com/docs/api/) to configure your GSLB. For a more visual experience, you can accomplish most of the same steps directly through the [ngrok dashboard](https://dashboard.ngrok.com/).
+
+Your first task is to [generate a new API key](https://dashboard.ngrok.com/api) in the ngrok dashboard. Make sure you save the API key before you close the modal, because it won't be displayed again.
+
+To simplify authenticating your account with the ngrok API, export the API key on your local workstation.
+
+```bash
+export NGROK_API_KEY=<YOUR-API-KEY>
+```
+
+Export a variable for your new domain, which will be used a few API calls below. With ngrok, you can reserve any subdomain that ends in an [ngrok-managed domain](https://ngrok.com/docs/network-edge/domains-and-tcp-addresses/) like `.ngrok.app`, `.ngrok.dev`, `.ngrok.pro`, or even `.ngrok.pizza`&mdash;for example, `linode-gslb-test.ngrok.app` might be an appropriate domain to reserve.
+
+Export a new variable for the domain of your choosing.
+
+```bash
+export NGROK_DOMAIN=<YOUR-NGROK-DOMAIN>
+```
+
+Create your domain on the [`/reserved_domains` endpoint](https://ngrok.com/docs/api/resources/reserved-domains/) using the `NGROK_API_KEY` and `NGROK_DOMAIN` variables you exported.
+
+```bash
+curl \
+  -X POST https://api.ngrok.com/reserved_domains \
+  -H "Authorization: Bearer ${NGROK_API_KEY}" \
+  -H "Content-Type: application/json" \
+  -H "Ngrok-Version: 2" \
+  -d '{"description":"Linode Load Balancing","domain":"'${NGROK_DOMAIN}'"}'
+```
+
+:::tip Custom domains within ngrok
+
+Using CNAMEs, ngrok can host an endpoint on your dowmain while also managing the complete TLS certificate lifecycle on your behalf. If you'd prefer to use a custom domain rather than an ngrok-managed one, [follow our guide](https://ngrok.com/docs/guides/how-to-set-up-a-custom-domain/) to set up your DNS and replace your custom domain in the `export` command above.
+
+:::
+
+## **Step 2**: Create your ngrok Edge {#create-edge}
+
+With your domain reserved, you can create an [ngrok Edge](https://ngrok.com/docs/network-edge/edges/) to manage all your endpoints and tunnels with a single configuration.
```

Comment:
This is a great question. I've always assumed some familiarity with ngrok for these how-tos, but I'm also open to this campaign requiring a slightly different approach. It can't hurt to add a little extra text, no?